android/engine/src/main/java/org/smartregister/fhircore/engine/data/local/DefaultRepository.kt

```diff
@@ -132,8 +179,7 @@ constructor(open val fhirEngine: FhirEngine, open val dispatcherProvider: Dispat
           fhirEngine.update(updateFrom(resource))
         }
       } catch (resourceNotFoundException: ResourceNotFoundException) {
-        resource.generateMissingId()
```

Comment:
Do we no longer need to this line `resource.generateMissingId()`?

Reply:
Yea, we no longer need that.

Since create() already has that.

Reply:
Noted.